src/MizarWidgets/MizarMainWindow.cpp

```diff
@@ -12,6 +12,10 @@ namespace orbit_mizar_widgets {
 
 MizarMainWindow::MizarMainWindow() : ui_(std::make_unique<Ui::MainWindow>()) { ui_->setupUi(this); }
```

Comment:
Sry. This is unrelated to this PR, nevertheless I wanted to mention that.

All classes that derive from QWidget should have a `QWidget* parent = nullptr` constructor parameter which is passed on to the base class.